Examples/instance-flow.md

```diff
@@ -1,61 +1,105 @@
+
+2. cURL commands in examples use -k parameter for convenience allowing curl to proceed and operate even for server connections considered insecure. Don't use it in production.
+
+3. When executing a command, it takes a while for the resources to be ready, and calling the next step before the previous is completed may lead to error (i.e. right after creating the provider it will start the synchronization, and the deployment box cannot be created until the first sync completes).
 
 Now let’s look at the script in sections to understand how you can make API calls from any code you like.
 
-### Authenticate with Cloud Application Manager
-Before calling to the API you have to sig in into the Cloud Application Manager website and [getting an authentication token](https://www.ctl.io/knowledge-base/cloud-application-manager/administering-your-organization/admin-access/). You use this token as an http header to perform every call to the Cloud Application Manager API.
+## Authenticate with Cloud Application Manager
+
+First of all, you need to hold the administrator role on a workspace to be able to run these steps. You must obtain [Administrator access](https://www.ctl.io/knowledge-base/cloud-application-manager/administering-your-organization/admin-access/) to Cloud Application Manager. 
+Before calling to the API you have to sign in into the Cloud Application Manager website and [get an authentication token](https://www.ctl.io/api-docs/cam/#getting-started).You will use this token as an authentication HTTP header to perform every call to the Cloud Application Manager API.
+
+## Obtain Deployment Variables
 
-**Declare Deployment Variables**
+Next, define the variables that will contain the parameters of the script. This way, each time you run the script, you can pass different deployment arguments. These declared variables are referenced later in your script as request parameter values.
 
-In this case we decided to use AWS so we will need the account key and secret to register it as provider. Box deployment arguments such as owner, environment and token are declared as variables because they can differ between deployments. That way, each time you run the script, you can pass different arguments.
+| Variable | Description |
+|----------|-------------|
+| provider_key | You need the provider account key and secret to register a provider. |
+| provider_secret | See below how to [register a provider](#Register-a-Provider-in-Cloud-Application-Manager) and get its parameters. |
+| json_web_token | This is the [authentication token](#Authenticate-with-Cloud-Application-Manager) obtained before.<br>It's a long BASE64 encoded string that starts with something like:<br/>  *eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJvcGVyYb25z...* |
+| environment | Set your URL organization hostname or your appliance IP, where API calls are sent.<br>Examples are:  *your-organization.cam.ctl.io*, *centurylink.cam.ctl.io*,  *192.168.1.10*. |
```

Comment:
Please remove the `centurylink.cam.ctl.io` becase `centurylink` is a valid CAM org. Better substitute it simply with `cam.ctl.io`.